snafu/uperf_wrapper/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM registry.access.redhat.com/ubi8:latest
 
 RUN dnf install -y --nodocs git python3-pip && dnf clean all
-COPY /src/image_resources/centos8-appstream.repo /etc/yum.repos.d/centos8-appstream.repo
+COPY snafu/image_resources/centos8-appstream.repo /etc/yum.repos.d/centos8-appstream.repo
```

Comment:
does this need the leading / that the old src line had? I dont think so but asking to be sure